app/client/components/create-volume-form.tsx

```diff
@@ -81,7 +85,14 @@ export const CreateVolumeForm = ({ onSubmit, mode = "create", initialValues, for
 	const handleTestConnection = async () => {
 		const formValues = getValues();
 
-		if (formValues.backend === "nfs" || formValues.backend === "smb" || formValues.backend === "webdav") {
+		if (
+			formValues.backend === "nfs" ||
+			formValues.backend === "smb" ||
+			formValues.backend === "webdav" ||
+			formValues.backend === "mariadb" ||
+			formValues.backend === "mysql" ||
+			formValues.backend === "postgres"
+		) {
```

Comment:
Maybe we could make this a constant `["nfs", "smb", ...]` and here refactor to if `SUPPORTS_CONNECTION.includes(formValues.backend) {}` as this condition is becoming to big